SPECS/kubernetes/kubernetes-1.17.11.spec

```diff
@@ -175,6 +183,10 @@ fi
 %{_bindir}/kubeadm
 
 %changelog
+* Tue Jan 05 2021 Nicolas Guibourge <nicolasg@microsoft.com> - 1.17.11-6
+- Fix test issue when building against golang 1.15
+- CVE-2020-8563
```

Comment:
Might be good to mention if it was fixed or nopatched. I know you can check that by viewing the rest of the spec, but why make people go back and forth through it?

Reply:
you can see it from the patch instruction (.nopatch extension)